packages/flutter/test/gestures/gesture_binding_resample_event_on_widget_test.dart

```diff
@@ -78,35 +81,39 @@ void main() {
       ),
     );
 
-    GestureBinding.instance!.resamplingEnabled = true;
-    const Duration kSamplingOffset = Duration(milliseconds: -5);
-    GestureBinding.instance!.samplingOffset = kSamplingOffset;
-    ui.window.onPointerDataPacket!(packet);
-    expect(events.length, 0);
+    await withClock(binding.clock, () async {
```

Comment:
As far as I understand this does similar, if not identical, job to `fakeAsync`, which is what Flutter has been doing. (I guess you had to use `withClock` because you added `clock` to `_Resampler`, isn't t?)

Reply:
The fakeAsync clock used for widget tests doesn't make it through to the code that is being tested, which is why I had to add this special hook here. I guess that can be fixed but probably best to keep the current behavior if using clock in framework code is not supported today.